deploy/playbooks/deploy.yml

```diff
@@ -7,6 +7,7 @@
     - celery
   vars:
      app_name: "mita"
+     prod_fqdn: "mita.ceph.com"
```

Comment:
I was more wondering if the deploy to vagrant would fail without that variable defined, this won't cause trouble for prod though so I'll merge.